### PR TITLE
Unfucks preview icons

### PR DESCRIPTION
**Pull request by ZomgPonies**
https://github.com/Baystation12/Baystation12/pull/9206

Unfucks preview icons so we never have to touch them again.

### CODE REVIEW

File: code/modules/mob/new_player/preferences_setup.dm [hunk 1]
```diff
@@ -190,613 +190,42 @@ datum/preferences
...
-
-		if(!current_species.bloodflags & BLOOD_SLIME)
-			preview_icon.Blend(eyes_s, ICON_OVERLAY)
-		if(underwear_s)
-			preview_icon.Blend(underwear_s, ICON_OVERLAY)
-		if(undershirt_s)
-			preview_icon.Blend(undershirt_s, ICON_OVERLAY)
-		if(clothes_s)
-			preview_icon.Blend(clothes_s, ICON_OVERLAY)
+	proc/update_preview_icon()		//seriously. This is horrendous. //less so now
```
Comment (Iamgoofball) [inline code/modules/mob/new_player/preferences_setup.dm]: >relative pathed

**Comment by ZomgPonies** [comment]
Hold off on this for a bit.

**Comment by ZomgPonies** [comment]
Its got a few crashy bugs that are reproduced on BS12. Hold on for a bit

**Comment by Regen1** [comment]
26 days and counting.
What is holding this back?

**Comment by ZomgPonies** [comment]
Client crashes are holding it back. Dunno if they are related to lighting, but until I get more info, I'm not getting this merged yet.

**Comment by Fox-McCloud** [comment]
Going to go ahead and close this; re-open it at a later date, if necessary once client crashes (or crashes related to this) cease and you can better pin things down.

**Pull request closed without merging**